pulp_ansible/app/__init__.py

```diff
@@ -9,3 +9,4 @@ class PulpAnsiblePluginAppConfig(PulpPluginAppConfig):
     name = "pulp_ansible.app"
     label = "ansible"
     version = "0.14.0.dev"
+    python_package_name = "pulp-python"
```

Comment:
Didn't know this package was also called pulp-python. TIL. :smiley:

Reply:
in https://github.com/ansible/galaxy_ng/pull/1314 we have `python_package_name = "galaxy-ng"` which matches pypi. Why does this not say `pulp-ansible` ... does it refer to https://pypi.org/project/pulp-python/?

Reply:
I'm pretty sure it was a copy-paste error, don't worry you are correct.

Reply:
lol yeah this needs to `pulp-ansible`